starknet_py/serialization/data_serializers/cairo_data_serializer.py

```diff
@@ -63,3 +65,7 @@ def serialize_with_context(
         :param value: python value to serialize.
         :return: defined SerializationType.
         """
+
+    @staticmethod
+    def remove_units_from_serialized_data(serialized_data: List) -> List:
+        return list(filter(lambda x: x is not None, serialized_data))
```

Comment:
```suggestion
        return [x for x in serialized_data if x is not None]
```
shorter + easier to read 😃